addons/test_event_full/tests/common.py

```diff
@@ -54,6 +54,16 @@ def setUpClass(cls):
             signature='--\nErnest',
         )
 
+        cls.user_event_web_manager = mail_new_test_user(
```

Comment:
is it really necessary to add this new user ? Does not seems we do something that should be impacted by the user, so we could probably use the `admin_user` perhaps